server/src/main/java/edu/colostate/cs/cs414/chesshireCoders/jungleServer/service/impl/RegistrationServiceImpl.java

```diff
@@ -39,7 +39,7 @@ public void registerUser(String nickName, String email, String hashedPassword) t
     }
 
     @Override
-    public void unregisterUser(String email, AuthToken token) throws SQLException {
+    public void unregisterUser(String email) throws SQLException {
```

Comment:
Would this allow an somebody to unregister a user that is not them?

Reply:
I'm thinking token checking will be done at the handler level. Otherwise it couples the service classes to each other.

It also made it easier for testing I think

Reply:
But I understand the concern